platforms/core-runtime/client-services/src/main/java/org/gradle/launcher/daemon/client/DaemonClientFactory.java

```diff
@@ -73,6 +74,6 @@ private DefaultServiceRegistry createLoggingServices(ServiceLookup clientLogging
      */
     public ServiceRegistry createMessageDaemonServices(ServiceLookup clientLoggingServices, DaemonParameters daemonParameters) {
         // These can always run inside the current JVM since we should not be forking a daemon to run them
-        return createBuildClientServices(clientLoggingServices, daemonParameters, new DaemonRequestContext(Jvm.current(), null, Collections.emptyList(), false, NativeServices.NativeServicesMode.NOT_SET, DaemonParameters.Priority.NORMAL), new ByteArrayInputStream(new byte[0]));
```

Comment:
Split `Priority` out of `DaemonParameters`, so they can move to different projects.